components/eamxx/tests/uncoupled/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ if (NOT SCREAM_BASELINES_ONLY)
   add_subdirectory(cld_fraction)
   add_subdirectory(spa)
   add_subdirectory(surface_coupling)
+  add_subdirectory(cosp)
```

Comment:
Shouldn't you put this inside the SCREAM_DOUBLE_PRECISION if, as you did in the src folder?